lib/foundation_rails_helper/form_builder.rb

```diff
@@ -18,15 +18,16 @@ class FormBuilder < ActionView::Helpers::FormBuilder
     end
 
     def label(attribute, text = nil, options = {})
-      if has_error?(attribute)
+      if error?(attribute)
         options[:class] ||= ''
         options[:class] += ' is-invalid-label'
       end
 
       super(attribute, (text || '').html_safe, options)
     end
 
-    def check_box(attribute, options = {}, checked_value = '1', unchecked_value = '0')
+    def check_box(attribute, options = {}, checked_value = '1',
+                  unchecked_value = '0')
```

Comment:
Yes, I didn't even know this was valid Ruby, but it worked.  However, I will use `rubocop:disable` on these definitions since we are overloading a Rails method so there is no good way to reduce length.